tests/Frameworks/Custom/Version_No_Autoloaded/index.php

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
Nice! Just a nit-pick - can we rename the containing folder to `Version_Not_Autoloaded`? :)